src/digraph.rs

```diff
@@ -179,6 +179,24 @@ impl NodeCount for PyDiGraph {
 
 // Rust side only PyDiGraph methods
 impl PyDiGraph {
+    fn _add_edge_no_checks(
+        &mut self,
+        p_index: NodeIndex,
+        c_index: NodeIndex,
+        edge: PyObject,
+    ) -> PyResult<usize> {
```

Comment:
```suggestion
    ) -> usize {
```
This function does not need to return a `Result` since it never actually fails.